LEAF_Nexus/templates/print_subindicators.tpl

```diff
@@ -27,7 +27,7 @@
             <!--{/if}-->
```

Comment:
I would go ahead and `strip_tags|escape` the following since they are being written to the DOM:
* the two `div` ids above
* the `$uid`, `$categoryID`, and `$indicator.indicatorID` in the two `getForm()` calls above